signup.html

```diff
@@ -25,23 +25,26 @@
           <div class="sign-input-box">
             <label for='email' class="text-medium">이메일</label>
             <input id='email' class="sign-input" />
+            <div id = "emailErrorMessage" class = "error-message text-medium hidden"></div>
           </div>
           <div class="sign-input-box sign-password">
             <label for='password' class="text-medium">비밀번호</label>
             <input id='password' class="sign-input" type="password" />
-            <button class="eye-button" type="button">
+            <div id = "passwordErrorMessage" class = "error-message text-medium hidden"></div>
+            <button id = "eye-button-password" class="eye-button" type="button">
               <img src="./assets/logo/eye-off.svg" />
             </button>
           </div>
           <div class="sign-input-box sign-password">
             <label for='password_comfirm' class="text-medium">비밀번호 확인</label>
             <input id='password_comfirm' class="sign-input" type="password" />
-            <button class="eye-button" type="button">
+            <div id = "passwordConfirmErrorMessage" class = "error-message text-medium hidden"></div>
+            <button id = "eye-button-password-confirm" class="eye-button" type="button">
               <img src="./assets/logo/eye-off.svg" />
             </button>
           </div>
         </div>
-        <button class="cta" type="submit">회원가입</button>
+        <button id="sign-in-button" class="cta" type="submit">회원가입</button>
```

Comment:
sign up 버튼이 되어어야 할 것 같네요ㅎㅎ
```suggestion
        <button id="sign-up-button" class="cta" type="submit">회원가입</button>
```

Reply:
Commit ID : 892b2c2e20bebdb31ec10e583b3be58c106a472a
정정했습니다 :D!